service/lib/agama/storage/proposal.rb

```diff
@@ -258,67 +188,39 @@ def storage_manager
         Y2Storage::StorageManager.instance
       end
 
-      # Returns an issue if there is no candidate device
+      # Returns an issue if there is no boot device.
       #
       # @return [Issue, nil]
-      def empty_candidate_devices_issue
-        return if !proposal || candidate_devices.any?
+      def boot_device_issue
+        return if settings.boot_device
 
-        Issue.new("No devices are selected for installation",
+        Issue.new("No device selected for installation",
```

Comment:
localization? As very soon (TM) dbus will be able to return localized string, I think that those errors will be useful.

Reply:
Yes. I would prefer to adapt all text as part of a separate issue once the localization is ready.